### PR TITLE
📝 Updated index page with new Defender description

### PR DESCRIPTION
**Pull request by cairoeth**

### CODE REVIEW

File: components/home/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,30 +1,25 @@
...
-xref:upgrades.adoc[[.card-title]#Upgrades# [.card-body]#pass:q[A comprehensive suite of contracts and tooling to deploy and manage upgradeable contracts on Ethereum.]#]
---
-
 [.card.card-primary.card-defender]
 --
-xref:defender::index.adoc[[.card-title]#Defender# [.card-body]#pass:q[A secure platform to automate Ethereum operations.]#]
+xref:defender::index.adoc[[.card-title]#Defender# [.card-body]#pass:q[A secure platform to code, audit, deploy, monitor, and operate your blockchain application.]#]
 --
 
-[.card.card-secondary.card-test-helpers]
```
Comment (frangio) [inline components/home/modules/ROOT/pages/index.adoc]: I removed Test Helpers. We're not actively maintaining them and they should not be featured prominently.

**Comment by frangio** [comment]
I updated styling of the cards to have 2 columns rather than 3.

**Review by frangio — DISMISSED** [review]
I would prefer to keep Upgrades as a primary card as it is an important part of our open source offering, but on the technical side this PR is okay.

**Comment by zanbel** [comment]
@cairoeth I see this issue has been stale for a while. Can we move forward and merge the text changes.

**Review by ernestognw — APPROVED** [review]
LGTM, thanks @cairoeth

**Pull request merged**